File: Python/Program/Human.py
class Human:
    __NIK = ""
    __name = ""
    __gender = ""

    # Konstruktor
    def __init__(self, nik = "", name = "", gender = ""):
        self.__NIK = nik
        self.__name = name
        self.__gender = gender
    
    # Getter and Setter #

    # Get NIK
    def get_nik (self):
        return self.__NIK
    
    # Get Name
    def get_name (self):
        return self.__name

    # Get Gender
    def get_gender (self):
        return self.__gender
    
    # Set NIK
    def set_nik (self, NIK):
        self.__NIK = NIK

    # Set Name
    def set_name (self, name):
        self.__name = name
    
    # set Gender
    def set_gender (self, gender):
        self.__gender = gender
    
    # Public Method #
    def printName (self):
        print("Name             : ", self.get_name())
    def printHuman(self):
        print("Name             : ", self.get_name())
        print("NIK              : ", self.get_nik())
        print("Gender           : ", self.get_gender())

File: C++/Program/Human.cpp
#include "Header.hh"

/* HUMAN SECTION */
Human::Human()
{
    nik = "";
    nama = "";
    gender = ""; 
}

Human::Human(string nik, string nama, string gender)
{
    this->nik = nik;
    this->nama = nama;
    this->gender = gender;
}

string Human::getNik()
{
    return this->nik;
}

void Human::setNik(string nik)
{
    this->nik = nik;
}

string Human::getNama()
{
    return this->nama;
}

void Human::setNama(string nama)
{
    this->nama = nama;
}

string Human::getGender()
{
    return this->gender;
}

void Human::setGender(string gender)
{
    this->gender = gender;
}

Human::~Human()
{

}